test/e2e/admin_test.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/openshift/openshift-azure/pkg/jsonpath"
 )
 
-var _ = BeforeSuite(func() {
-	c = newTestClient(*kubeconfig)
```

Comment:
This needs to be added back.

Reply:
This got moved to `extended_suite_test.go:BeforeEach` (along with cluster creation) do you feel the client should be created only in `BeforeSuite`?

Reply:
Eventually we may want to have separate clients per namespace, doesn't seem like we gain something by doing it now.